nixos/modules/services/x11/display-managers/gdm.nix

```diff
@@ -52,6 +52,14 @@ in
         debugging messages in GDM
       '';
 
+      forceGnomeSessions = mkOption {
```

Comment:
Missed this when I looked over last time.

I'm not really convinced this option is necessary tbh. We already have an option to enable gnome sessions through the gnome desktop, so this seems a bit superfluous.

Reply:
I thought this could be a way to restore the current behavior in case somebody would need it (even if broken). Was not convinced either tbh : I removed it.